packages/compass-explain-plan/src/components/explain-json/explain-json.tsx

```diff
@@ -0,0 +1,39 @@
+import type { FunctionComponent } from 'react';
+import { useCallback, useMemo } from 'react';
+import React from 'react';
+import HadronDocument from 'hadron-document';
+import { Document } from '@mongodb-js/compass-crud';
+import { css, KeylineCard, spacing } from '@mongodb-js/compass-components';
+
+const cardStyles = css({ padding: spacing[2] });
+interface ExplainJSONProps {
+  originalExplainData: Record<string, unknown>;
+}
+
+const ExplainJSON: FunctionComponent<ExplainJSONProps> = ({
+  originalExplainData,
+}) => {
+  const copyToClipboard = useCallback(() => {
+    void navigator.clipboard.writeText(JSON.stringify(originalExplainData));
+  }, [originalExplainData]);
+
+  const doc = useMemo(
+    () => new HadronDocument(originalExplainData),
+    [originalExplainData]
+  );
+
+  return (
+    <KeylineCard data-testid="explain-json" className={cardStyles}>
+      <Document
+        copyToClipboard={copyToClipboard}
+        doc={doc}
+        editable={false}
+        isExpanded={false}
+      />
+    </KeylineCard>
+  );
+};
+
+ExplainJSON.displayName = 'ExplainJSONComponent';
```

Comment:
If we are rewriting these, can we get rid of this explicit display name setting? It's pretty useless

Reply:
that would be great, but i think `eslint` complains about it, not sure why, maybe depends on how I define it, I'll try though

Reply:
ok no, I lied, probably the first time I converted it in jsx or something or in a different way, anyway it doesn't complain

Reply:
Ahh, you know, eslint complaining might be the case for this other place where you use `forwardRef`, because the function inside `forwardRef` is anonymous. Anyway, it's super small thing, so whatever works here 😃